src/Ting/Query/Generator.php

```diff
@@ -169,17 +170,29 @@ public function getOneByCriteria(
     /**
      * @param array           $criteria
      * @param DriverInterface $driver
+     * @param array      $order
+     * @param int        $limit
      * @return array
      */
-    protected function getSqlAndParamsByCriteria(array $criteria, DriverInterface $driver)
+    protected function getSqlAndParamsByCriteria(array $criteria, DriverInterface $driver, array $order = [], $limit = 0)
```

Comment:
This method is not marked as internal, so its a BC break when someone extends this class : https://3v4l.org/WJ2Wv